src/dshould/basic.d

```diff
@@ -307,8 +307,8 @@ if (isInstanceOf!(ShouldType, Should)
 
         check(
             mixin(format!(enums.checkString)("got", "expected")),
-            format("value %s", enums.message.format(expected.quote)),
-            format("%s", got.quote),
+            enums.message.format(expected.quote),
```

Comment:
Why not `format!(enums.message)(expected.quote)`?
The "new" `format` template is already used in the line above.
Otherwise, I still would prefer `format(enums.message, expected.quote)`

Reply:
`format(enums.message, expected.quote)` done.
I'm trying to avoid prompting format to call toString on null objects, as it does on older DMD versions. This can lead to problems if the null state is invalid.